src/js/worklets/parallax.js

```diff
@@ -0,0 +1,41 @@
+import {
+    useAnimatedStyle,
+    useAnimatedSensor,
+    withTiming,
+    interpolate,
+    SensorType
+} from 'react-native-reanimated';
+import { Platform } from 'react-native';
+
+const PI = Math.PI;
+
+export function sensorAnimatedImage(zIndex, offset, stretch) {
+    'worklet'
+    const sensor = useAnimatedSensor(SensorType.ROTATION, { interval: 30 })
+    return useAnimatedStyle(function () {
+
+        const { qx, qz, qw, qy, pitch } = sensor.sensor.value;
+
+        const roll = Math.asin(-2.0 * (qx * qz - qw * qy))
+
+        const top = withTiming(
+            interpolate(
+                pitch,
+                Platform.OS === 'ios' ? [-PI, PI] : [PI, -PI],
+                [(-offset * 3) / zIndex - offset + (offset - stretch), (offset * 3) / zIndex - offset + (offset - stretch)]),
+            { duration: 10 }
+        );
+        const left = withTiming(
+            interpolate(
+                roll,
+                [1, -1],
+                [(-offset * 2) / zIndex - offset + (offset - stretch), (offset * 2) / zIndex - offset + (offset - stretch)]),
+            { duration: 10 }
+        );
+
+        return {
+            transform: [{ translateX: left },
+            { translateY: top }]
```

Comment:
Maybe improve indentation / code styling, though we don't have definitions for JavaScript I think.

I'd suggest 
```
transform: [
  { translateX: left },
  { translateY: top }
]
```

Reply:
cool will do, yeah I think we need to add prettier or something for js

Reply:
Its worth having a prettier config and a make command that runs it @J-Son89.
Not in this PR obviously :)